docs/src/main/sphinx/connector/non-transactional-insert.fragment

```diff
@@ -0,0 +1,13 @@
+Non-transactional INSERT
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+The connector supports adding rows using :doc:`INSERT statements </sql/insert>`.
+By default, data insertion is performed by writing data to a temporary table.
+You can skip this step to improve performance and write directly to the target
+table. Set the catalog property ``insert.non-transactional-insert.enabled``
+or the corresponding catalog session property ``non_transactional_insert`` to
+``true``.
+
+In rare cases when exceptions occur during the insert operation, data in
```

Comment:
Prefix with "When this property is enabled". This issue is non-existent in the default config.